planning/development-strategy.md

```diff
@@ -25,86 +25,161 @@ A simple little website about trees.
 
 <!-- issue title -->
 
-## Site Title
+## Site Title (milestone)
 
 <!-- user story -->
 
-> "as a user I can read the title of the page"
+> "As a user I can read the title of the page"
 
 <!-- detailed description -->
 
-The site needs a clear title that's easy to read at the top of the page.
+<!--The site needs a clear title that's easy to read at the top of the page.-->
 
 <!-- acceptance criteria -->
 
 - [ ] _there is a prominent title at the top of the web page_
 
 <!-- code you think you will need -->
 
-### Site Title: HTML
+### Site Title HTML
 
-- Header with the web site's title
+- Header with the web site's title (issue)
 
-### Site Title: CSS
+### Site Title CSS
 
-- styling to center the title
+- styling to center the title(issue)
 
 ---
 
-## Introduction
+## Menu (milestone)
 
-> "As a user I can read an introduction to this project"
+> As a user I can read the details of the website from the Menu.
 
-It should be clear what this project is for and what someone can expect from the
-web page.
+- [ ] _A Menu contians Home,Introduction,Content and Extras are displayed at the
+      top of the page._
 
-- [ ] _there is a section directly under the navbar describing the project's
-      goals_
+### Menu HTML
+
+-[ ] Navbar:Home,Introduction,Content,Extras(issue)
+
+### Menu CSS
+
+- [ ] style elements, hover (issue)
+- [ ] place elements using flex (issue)
+
+---
+
+## Introduction (milestone)
+
+> "As a user I can read an introduction to this project" It should be clear what
+> this project is for and what someone can expect from the web page.
+
+- [ ] _there is a section "Introduction" in the navbar describing the project's
```

Comment:
What do you mean by Introduction here? If the link in the menu, then you have already described it above.
We don't need to describe the same feature twice.

Reply:
One of the menu link is introduction. Menu / navbar contains introduction, content and extras

Reply:
### Menu HTML

-[ ] Navbar:Introduction,Content,Extras(issue)

I just mentioned the menu elements here. so after that a well explained introduction.